### PR TITLE
Fixes #84

### LINKED ISSUE #84: Add move_hand function from kuka
As demanded by Armando, the commander tool created for the kuka arm + hand to make it easier to write scripts for the hand should be included in the standard package for the hand.

Basically it adds interpolation and allows you to determine how long will it take for the hand to go from the current pose to the commanded pose.

It's the move_hand function in the file:

https://github.com/shadow-robot/kuka/blob/development/kuka_commander/src/kuka_commander.py

### PR DESCRIPTION
**Pull request by toliver**
To test it run:

roslaunch sr_hand gazebo_arm_and_hand.launch
rosrun sr_example eg_hand_commander

### CODE REVIEW

File: sr_hand/scripts/sr_hand/shadowhand_commander.py [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+
+"""
+Module to provide a quick and easy way to script the Shadow Hand.
+This is done via creating a short script with a Commander object then calling
+methods on the commander to script the motion.
+
+Basic script will look like:
+
+    #!/usr/bin/python
+    import roslib;
```
Comment (markpitchless) [inline sr_hand/scripts/sr_hand/shadowhand_commander.py]: Not needed, import rospy should be enough

File: sr_hand/scripts/sr_hand/shadowhand_commander.py [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+        to their previous targets at their previous velocity.
+
+        @param joints - Dictionary of joint names in the keys and angles in
+        degrees in the values. The key interpolation_time gives the time in seconds that 
+        the movement will last.
+        """
+        
+        interpolation_time = 0.0
+        if 'interpolation_time' in joints:
+            interpolation_time = joints['interpolation_time']
```
Comment (markpitchless) [inline sr_hand/scripts/sr_hand/shadowhand_commander.py:96]: Should possible remove 'interpolation_time' from the dict here just to keep the joints dict clean, ie only joints?

Comment (toliver) [inline sr_hand/scripts/sr_hand/shadowhand_commander.py:96]: he he. Yes. ;)

**Comment by toliver** [comment]
Done

**Pull request merged**